sys/sys/pcpu.h

```diff
@@ -59,7 +59,7 @@
  */
 extern uintptr_t *__start_set_pcpu __no_subobject_bounds;
```

Comment:
I still don't understand why these aren't using `[]`...

Reply:
That would be a change to make upstream both for here and vnet.h?

Though really these should be using `SET_DECLARE` I think from <sys/linker_set.h>.  Note that __MAKE_SET_QV there already uses __nosanitizeaddress and perhaps it should have a __no_subject_bounds annotation as well.  Note though that DECLARE_SET() doesn't declare as arrays but as pointers.